src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardHelper.js

```diff
@@ -233,12 +231,8 @@
                             options: currentField.options
                         }
 
-                        if (currentField.sObjectName === 'Opportunity' &&
```

Comment:
Should this be removed here or is this carried over from the locked date branch?